polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -1833,7 +1833,10 @@ impl<T: Config> Pallet<T> {
 				// no custom fees instructions, they are batched together with `assets` transfer;
 				// BuyExecution happens after receiving all `assets`
 				let reanchored_fees =
-					fees.reanchored(&dest, &context).map_err(|_| Error::<T>::CannotReanchor)?;
+					fees.reanchored(&dest, &context).map_err(|e| {
+						log::error!(target: "xcm::pallet_xcm::add_fees_to_xcm", "Failed to re-anchor fees: {:?}", e);
```

Comment:
I would also log here (whatever is possible without additional cloning), e.g.: adding `&dest, &context` makes it much more useful log, so you can take those exact values from the log and play with unit-test or whatever
```suggestion
						log::error!(target: "xcm::pallet_xcm::add_fees_to_xcm", "Failed to re-anchor fees: {e:?} for dest: {dest:?} and context: {context:?}");
```

@ayevbeosa and also please check/fix all other `reanchored` here

Reply:
All done!